lib/orbit/ocs/stream/pipeline.ex

```diff
@@ -41,14 +44,21 @@ defmodule Orbit.Ocs.Stream.Pipeline do
       records
       |> Enum.flat_map(&parse_records/1)
       # |> update_ocs_sequence_monitor
-      |> Enum.map(&ocs_message/1)
+      |> Enum.flat_map(&ocs_message/1)
```

Comment:
question: would you mind providing the backstory on this?

Reply:
As per our conversation, this was an (arguably confusing) way to filter out non-ocs messages. I put up a revised version in 663f02f that I think is more clear.